setup.py

```diff
@@ -79,7 +79,12 @@
         'console_scripts': [
             'legit = legit.cli:main',
         ],
-    }
+    },
+    data_files=[
```

Comment:
Except @wkentaro mentioned the issue with macOS system path `/System/Library/Frameworks/Python.framework/versions/2.7`, if someone wants to install `legit` as `/usr/bin/legit`, then data files are installed as `/usr/man`, `/usr/etc`, breaking FHS.

man and etc paths should be different based on `sys.prefix`.